tests/test_awsdatabricks.py

```diff
@@ -610,7 +618,106 @@ def test_create_prediction_for_run_success(respx_mock):
             "ContentLength": 0,
             "Body": StreamingBody(io.BytesIO(), 0),
         },
-        {"Bucket": "bucket", "Key": f"{s3_file_prefix}/eventlog"},
+        {"Bucket": "bucket", "Key": f"{eventlog_file_prefix}/eventlog"},
+    )
+
+    def client_patch(name, **kwargs):
+        match name:
+            case "s3":
+                return s3
+            case "ec2":
+                return ec2
+
+    with s3_stubber, ec2_stubber, patch("boto3.client") as mock_aws_client:
+        mock_aws_client.side_effect = client_patch
+        result = create_prediction_for_run("75778", "Premium", "Jobs Compute", "my-project-id")
+
+    assert not result.error
+    assert result.result
```

Comment:
The exclusivity of result & error is enforced by a validator in the Response class

Reply:
ah - yeah, this was a little overzealous asserting, haha